Dockerfile

```diff
@@ -1,23 +1,36 @@
-FROM registry.opensource.zalan.do/stups/python:3.5-cd26
+FROM registry.opensource.zalan.do/stups/python:3.6.2-14
```

Comment:
latest version of stups/python  is 3.6.3-15 published 42h ago

Reply:
indeed lets do this with a new ticket: https://github.com/zalando/ghe-backup/issues/78